Assets/Scripts/GameController.cs

```diff
@@ -1,10 +1,185 @@
 ﻿using UnityEngine;
+using System;
+using System.Collections.Generic;
+
 
 public class GameController : MonoBehaviour {
 
+	private Token[][] _tokens;
+	private int _gridHeight;
+	private int _gridLength;
```

Comment:
Not too important, as long as it's consistent, but Width here instead of Length?